### PR TITLE
Barrels imp

### PR DESCRIPTION
**Pull request by brauliodiez**
Using ES6 libraries (adding alias on webpack) we get barrels + tree shaking.

### CODE REVIEW

File: typescript/03 barrels/readme.md [hunk 1]
```diff
@@ -1,79 +1,85 @@
...
+npm install material-ui@next --save
 ```
 
-- Let's install the typings for this library:
+- Let's uninstall the typings (@next already contains the typings in it)
 
-```
-npm install @types/material-ui --save-dev
+```bash
+npm install material-ui@next --save
```
Comment (nasdan) [inline typescript/03 barrels/readme.md]: Add command to `uninstall` typings package

File: typescript/03 barrels/src/app.tsx [hunk 2]
```diff
@@ -27,14 +25,15 @@ export class App extends React.Component<Props, State> {
   public render() {
     return (
       <>
-        <MuiThemeProvider>
```
Comment (nasdan) [inline typescript/03 barrels/src/app.tsx:30]: Dont remove `MuiThemeProvider`

Comment (brauliodiez) [reply typescript/03 barrels/src/app.tsx:30]: I think on @next is neeeded to be removed if you are not setting your custom them

File: typescript/03 barrels/readme.md (hunk 1 above)
Comment (brauliodiez) [reply typescript/03 barrels/readme.md]: Thanks I have updated that

File: typescript/03 barrels/src/app.tsx (hunk 2 above)
Comment (nasdan) [reply typescript/03 barrels/src/app.tsx:30]: Ahh ok, so we need to update Readme

Comment (nasdan) [reply typescript/03 barrels/src/app.tsx:30]: Nvm, it's updated. With all `+ -` it's difficult to review it

**Pull request merged**